### DISCUSSION TITLE
is there a way to fix the capture difference?

### DISCUSSION THREAD

**Discussion by 111100001** [MrBrax/LiveStreamDVR#513] [General]
just captured my first stream and found out there was a capture difference. why did that happen and is there a way to fix it? is it one whole chunk (in my case 20 minutes) or is it just bits and pieces totaling 20 minutes not captured through out the entire stream? the beginning and end of stream are there so it must be some where in the middle not captured.

also is there a way to access the captured chat and open it? i want to use ctrl+F to see if somethings were said during stream or other info

**Comment by MrBrax** [comment]
Capture differences are most likely ad segments, you can see it a little more specific under **Capture** -> **Stream pauses** _(Stream pauses: 00:45:08 (23)_ )

you can open the chatdump file in any text editor as long as it's a reasonable size, when it goes over a few dozen megabytes it will get very slow.

**Reply by 111100001** [reply]
thank you! i didnt think about the ads

the chat dump cannot be accessed from the web app am i right or i didn't look hard enough?

**Reply by MrBrax** [reply]
not directly, but you can expand the advanced options on the vod, right click the json button, copy link, and change out `.json` to `.chatdump`

**Reply by 111100001** [reply]
thank you very much
